### PR TITLE
issue #252 change view

### LINKED ISSUE #252: Fix teams displaying in playoff.
After the first round, in games, where one participant is known, another is diplayed as "Team #". Instead it must be 
"Team that known"
"Winnner#".

### Subtasks

- [x] #305  change view

- [x] #306 manual testing

### PR DESCRIPTION
**Pull request by BenRoman**
### Summary

Changed View
removed many "if-else" because our model already has away and a home team names
Closes #252 

#### Design

- [ ] Web API layer contains as little logic as possible
- [ ] Domain objects use semantic names

#### Perfromance

- [ ] In case of any DB query was changed: attach generated SQL for review
- [ ] Number of DB roundtrips should be as low as possible

#### Unit Tests

- [ ] Naming: Initial state and expected result are properly stated
- [ ] Test follows [AAA](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) pattern.
- [ ] Test is Isolated
- [ ] Mock instances and expected instances do not share references
- [ ] When new property is added all related comparers are updated

### CODE REVIEW

File: src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml [hunk 1]
```diff
@@ -48,230 +56,88 @@
...
             }
-        }
+        </div>
 
         if (Model.AllowedOperations.IsAllowed(AuthOperations.Games.Edit))
         {
             <div class="row">
                 <div class="col-md-2">
-                    @Html.ActionLink(Resources.UI.TournamentViews.Edit, "EditScheduledGame", new { gameId = Model.Id })
+                    @Html.ActionLink(Resources.UI.TournamentViews.Result, "Edit", "GameResults", new { Id = Model.Id }, null)
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml]: Why this one was changed?

**Comment by BenRoman** [comment]
We have already tested that 
It seems that all is OK

**Comment by sdiachen** [comment]
So what have you changed here and why there are so many man changes? How can we be sure we won't overwrite Smith important

File: src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml (hunk 1 above)
Comment (Myroslavv) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml]: image

Comment (Myroslavv) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml]: It's a little bit lower the page. Which is why I agree with Roman, that GitHub mistakenly shows this as editing, considering the fact that a lot of contitional statements were removed.

**Pull request merged**